php/class-customize-concurrency.php

```diff
@@ -14,4 +14,204 @@
  */
 class Customize_Concurrency {
 
-}
+	/**
+	 * Post type that stores settings changes until session ends.
+	 *
+	 * Limited by WP to 20 characters
+	 *
+	 * @var string $POST_TYPE
+	 */
+	const POST_TYPE = 'custom_saved_setting';
```

Comment:
Sure, `customize_c9y` works 😄 

After all, these posts are really transient and they could be deleted at any time without much consequence.